docs/status.md

```diff
@@ -1,20 +1,41 @@
 # Project status
 
-We’re really early on with the project and as always with any young project there are probably some sharp corners. But with the help of the open source community we plan to iron out those in the coming months.
+To gauge the level of interest in running k0s control planes as Kubernetes
+resources, we are releasing k0smotron at the earliest possible moment. Of
+cousre, as with any young project just getting started, there are bound to be
+some bumps, unforeseen issues, and a few surprises. In line with that, we can
+offer no guarantees regarding backward or forward compatibility, and you should
+expect a certain amount of breakage with each ensuing k0smotron release.
+
+All caveats aside, though, we are truly excited about what k0smotron has to
+offer, and look forward to seeing how it can transform the way you manage your
+Kubernetes deployments. We are confident that the open source community will
+take great interest in the project, too, and help us to smooth out the rough
+edges in the days, weeks, and months ahead.
 
-At this point we can not give much guarantees over backwards and forwards compatibility so expect things to break a bit with upcoming releases of k0smotron.
+## Cluster API
 
-We wanted to release k0smotron as early as possible to see whether there is in general interest of running k0s control planes as Kubernetes resources. We are excited about the potential that k0smotron brings to the table, and we look forward to seeing how it can transform the way you manage your Kubernetes deployments.
+One key focus of the k0smotron project is to get it to function as a Cluster
+API provider for both `ControlPlane` and worker `Bootstrap` providers. Once
```

Comment:
Currently, k0smotron can act as `ControlPlane`, `Bootstrap`, and `Infrastructure` provider. 
"(within the management cluster)" is redundant, with k0smotron currently we can provision the control plane both inside and outside the management cluster